operations/operations/handbook/README.md

```diff
@@ -0,0 +1,45 @@
+# Overview
+
+Welcome to the Mattermost Handbook process, policy, and onboarding section.
+
+As a remote-first company, we rely heavily on shared knowledge. Often, that knowledge is stored in peoples' heads, on a notepad somewhere, or in a Google Doc. The purpose of the Handbook is to capture everyone's knowledge of how Mattermost works so that we can all have access to it. This means that there's no ambiguity or confusion. It also means that when new staff members join us, it's easier for them to ramp up.
+
+So, let's get started!
+
+## Handbook-first
+
+As we mature, a Handbook-first policy is critical. But what does this mean?
+
+Handbook-first means that when you have a new idea, a new process, an update to a process, a change in your team, etc you document it in the Handbook first. For example, when a new staff member joins us, they should be added to your team's page before their first day. Another example is if you've created a new Jira process - first document it in the Handbook and then announce it to your team/the company. This allows you to find blindspots and also means that people can go to the relevant page immediately after the announcement to read more.
+
+### How do we drive Handbook-first?
+
+Great question. It's a team effort across the whole company, but we have a secret weapon in the form of Handbook Champions.
+
+## Handbook Champions
+
+A Handbook Champion is a volunteer from a specific team/operational area who drives Handbook-first adoption and acts as a go-to in the team for Handbook questions. While they aren't responsible for a set num
```

Comment:
this sentence appears to need an ending :)

Reply:
That definitely seems like the case. :D